orchestration/scripts/generate-staging-table.sh

```diff
@@ -0,0 +1,51 @@
+set -euo pipefail
```

Comment:
Can't comment on the file name directly, but I'd change it to be less generic. This script is specifically for generating an update/append staging table for ClinGen.